testsuite/tests/weak-ephe-final/weaktest_par_load.ml

```diff
@@ -1,4 +1,12 @@
-(* TEST *)
+(* TEST
+ no-tsan; (* Unsynchronized Weak.set and Weak.get is a data race (TODO #XXXX) *)
+ flags = "-g";
```

Comment:
Again, is this flag needed?

Reply:
This must be a left over from debugging. Also the `no-tsan` is no longer necessary as this test passes.
Fixed in 4fe058fd1c